Component/Products/product.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initiak-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>e-shop - products | e-commerce website</title>
```

Comment:
This is a different title than that is present  on the homepage

Reply:
yes. It should be different na for different pages? Isn't it?

Reply:
Yeah